python/GangaLHCb/Lib/Server/DiracLHCbDefinition.py

```diff
@@ -4,10 +4,49 @@
 import datetime
 import glob
 import pickle
+from funcutils import wraps
 ## NB parseCommandLine first then import Dirac!!
 from DIRAC.Core.Base.Script import parseCommandLine
 parseCommandLine()
 from LHCbDIRAC.Interfaces.API.DiracLHCb import DiracLHCb
 from DIRAC.Interfaces.API.DiracAdmin import DiracAdmin
 dirac = DiracLHCb()
 
+def diracCommand(f):
```

Comment:
Unless we move the definition into the Commands which I'm not sure are (or are needed to be) loaded when performing a BKQuery operation.